tests/cnf/ran/ptp/internal/consumer/bothversions.go

```diff
@@ -0,0 +1,99 @@
+package consumer
+
+import (
+	"fmt"
+
+	"github.com/rh-ecosystem-edge/eco-goinfra/pkg/clients"
+	"github.com/rh-ecosystem-edge/eco-goinfra/pkg/ptp"
+	. "github.com/rh-ecosystem-edge/eco-gotests/tests/cnf/ran/internal/raninittools"
+	"github.com/rh-ecosystem-edge/eco-gotests/tests/cnf/ran/internal/ranparam"
+	"github.com/rh-ecosystem-edge/eco-gotests/tests/cnf/ran/internal/version"
+)
+
+type ptpEventAPIVersion string
+
+const (
+	eventAPIVersionV1 ptpEventAPIVersion = "1.0"
+	eventAPIVersionV2 ptpEventAPIVersion = "2.0"
+)
+
+// DeployConsumersOnWorkers deploys the cloud-event-consumer on all worker nodes. It checks the event API version based
+// on the PTP version and event version in the PtpOperatorConfig then delegates to either [DeployV1ConsumersOnWorkers]
+// or [DeployV2ConsumersOnWorkers] to deploy the consumers.
+func DeployConsumersOnWorkers(client *clients.Settings) error {
```

Comment:
Just curious - typically for MNO, should PTP running on all nodes or just workers? I know that for ARM, we can only test on worker due to HW limitation.